cvmfs/network/download.cc

```diff
@@ -70,6 +70,31 @@ using namespace std;  // NOLINT
 
 namespace download {
 
+Tube<DataTubeElement>* DownloadManager::data_tube_empty_elements_ =
+                                              new Tube<DataTubeElement>(200000);
+atomic_int32 DownloadManager::counter_use_data_tube_ = 0;
+
+static DataTubeElement* GetUnusedDataTubeElement() {
+  DataTubeElement* ele = DownloadManager::data_tube_empty_elements_->
+                                                                  TryPopFront();
+
+  if (ele == NULL) {
+    char *data = static_cast<char*>(malloc(CURL_MAX_HTTP_HEADER));
```

Comment:
You have `CURL_MAX_WRITE_SIZE` down below, which seems like the more correct value.

Reply:
thanks thats an old left over. it should be everywhere `CURL_MAX_WRITE_SIZE`